content/project-organization/quality-council/processes/pr-management.md

```diff
@@ -0,0 +1,159 @@
+
+
+![img.png](images/readme10.png)
+
+
+For the color code, we always use **violet** for the **Branch** and this time we'll use **green** for the **PR number**. 
+
+
+We can find this **number** in the title of the PR, as you can see. 
+
+
+![img.png](images/readme11.png)
+
+
+All you have to do now is try to reproduce the issue again to see if the PR corrects it correctly.
+
+### 4. Check that the PR doesn't create another issue
+
+Now that you've checked that the PR has corrected the issue, you need to make sure that it doesn't create another one instead. To do this, you need test axes. 
+
+To find them, go to the Files changed tab: 
+
+![img.png](images/readme12.png)
+
+You'll then have all the changes made to the PR. To do this, you'll get this view: 
+
+![img.png](images/readme13.png)
+
+Seen like this, it may seem rather difficult to see the differences and what might be impacted, so I advise you to modify the split view and save.
+
+![img.png](images/readme14.png)
+
+You now have a better view of the various modifications, and can read the whole thing. 
+
+![img.png](images/readme15.png)
+
+No computer skills? But you still need to check that it's not overflowing? There are two things you can do: 
+
+- From which **file** the modification was made. You can see on the screen the name of the **file** framed in **Orange**. By seeing the names, I can tell that it will be on the front by the name "FrontController".
+- Thanks to the **comments**. These are framed in **blue**, meaning that there won't necessarily be a **comment** in each PR. You just need to read them to find out what the changes are and see what incidents they might create.
+
+Following this search and the various checks, you should now have the results of the automated tests. If there are any fails, you can re-run the tests as indicated in the automated test tips. 
+If you still have so many failures on the automated tests, you can test them manually on your shop. 
+
+
+### 5. Responding to contributors
+
+
+You've just finished testing the PR, and you're in two situations: 
+
+
+- The PR is valid, it doesn't create any other issue. In this case, I advise you to indicate that it works, to put a video proving it (you may think it's the desired behavior but it's not) and to put the link to the automated tests.
+In your comment, you'll need to select the Approve option before submitting the review.
+    
+![img.png](images/readme16.png)
+    
+- The PR is not valid. In this case, indicate that it doesn't work, put a video showing that it doesn't work and put the link to the automated tests (to prove that it brings issues). 
+In your comment, you'll need to select the Comment option before submitting the review. The Request changes option doesn't work properly.
```

Comment:
Is this part still up to date?

Reply:
Don't know, I continue to apply this method because if we don't, we have to accept the PR at the end, another QA can't approve it